subprojects/build-cache/src/main/java/org/gradle/caching/internal/origin/OriginMetadataFactory.java

```diff
@@ -56,28 +55,28 @@
     private final String operatingSystem;
     private final Clock clock;
     private final GradleVersion gradleVersion;
-    private final BuildInvocationScopeId buildInvocationScopeId;
+    private final UniqueId currentBuildInvocationId;
     private final File rootDir;
 
-    public TaskOutputOriginFactory(Clock clock, InetAddressFactory inetAddressFactory, File rootDir, String userName, String operatingSystem, GradleVersion gradleVersion, BuildInvocationScopeId buildInvocationScopeId) {
```

Comment:
Why this change? 

Using the stronger type makes viewing usages much easier.

Reply:
Do you mean the change from `BuildInvocationScopeId` to `UniqueId`? It's because `BuildInvocationScopeId` is defined in `:core` and is not strictly needed here.